src/site/layouts/va_form.drupal.liquid

```diff
@@ -1,7 +1,7 @@
 {% include "src/site/includes/header.html" with drupalTags = true %}
 {% include "src/site/includes/alerts.drupal.liquid" %}
 {% include "src/site/includes/preview-edit.drupal.liquid" %}
-{% include "src/site/includes/breadcrumbs.drupal.liquid" %}
+{% include "src/site/includes/breadcrumbs.drupal.liquid" with pathBreadcrumb = true %}
```

Comment:
`pathBreadcrumb` is a little unclear - maybe `deriveBreadcrumbsFromUrl`, `deriveBreadcrumbsFromPath`, or something

Reply:
Agreed, @ncksllvn